flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX == 0)
```

Comment:
The special meaning of `0` for `GameSizeX ` and `GameSizeY` should be documented in the `@param` docs.